browser/ui/content_settings/brave_content_setting_image_models.cc

```diff
@@ -7,30 +8,35 @@
 #include "brave/browser/ui/content_settings/brave_autoplay_blocked_image_model.h"
 #include "third_party/widevine/cdm/buildflags.h"
 
-#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
+#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT) || BUILDFLAG(BUNDLE_WIDEVINE_CDM)
 #include "brave/browser/ui/content_settings/brave_widevine_blocked_image_model.h"
 #endif
 
 void BraveGenerateContentSettingImageModels(
-    std::vector<std::unique_ptr<ContentSettingImageModel>>& result) {
+    std::vector<std::unique_ptr<ContentSettingImageModel>>* result) {
+  // lint complains to use pointer or const referencc for |result|.
```

Comment:
`*` is always preferable to `&` because it's more clear to the caller that you're passing something which might be modified when the method returns. If you're looking at a caller using ref, it's indistinguishable from passing be const ref or value unless you look at the method sig

Reply:
Yes, total agree about that.

Reply:
basically, this rule is fixed in CC, so we don't even need to have comments like this in the codebase :)

Reply:
Right. using like this is natural in chromium project :) Removed.